eclair-core/src/test/scala/fr/acinq/eclair/TestConstants.scala

```diff
@@ -66,7 +66,7 @@ object TestConstants {
       paymentRequestExpiry = 1 hour,
       maxPendingPaymentRequests = 10000000)
 
-    def id = nodeParams.privateKey.publicKey
+    def id = nodeParams.nodeId
```

Comment:
This should be removed and tests would use `Alice.nodeParams.nodeId` (some do already).